.gen/pipeline/api/openapi.yaml

```diff
@@ -24124,6 +24127,9 @@ components:
           type: string
         required:
           type: boolean
+        isPassword:
+          nullable: true
+          type: boolean
```

Comment:
**Question**: what triggered these changes?
Have you edited the generated file instead of the manual one?

Reply:
No, `generate-openapi` task (from Makefile) updated these files in `.gen` directory

Reply:
I understood that, but what was the source.

Isn't this change supposed to be coming from apis/pipeline/pipeline.yaml only?

Reply:
I could find it in the `apis/pipeline/pipeline.yaml` now, so it's fixed, I must have missed it on first look I guess.